joomla/joomla-cms
# [4.0] Content Languages action button

### Summary of Changes
Convert the toolbar buttons in the Content Languages page to use the same actions button as other components


### Testing Instructions
Check that all actions work as expected
Check that the empty trash button appears and works as expected when the filter is set to trashed
